frontend/packages/metal3-plugin/src/selectors/index.ts

```diff
@@ -42,3 +42,10 @@ export const getHostBios = (host: K8sResourceKind) => _.get(host, 'status.hardwa
 
 export const getHostMachine = (host: K8sResourceKind, machines: MachineKind[] = []) =>
   machines.find((machine: MachineKind) => getHostMachineName(host) === getName(machine));
+
+export const getMachineAddresses = (machine: MachineKind) => _.get(machine, 'status.addresses', []);
```

Comment:
Since these are machine related, it would be better to move them into `@console/shared/src/selectors/machine.ts` and import them from `@console/shared`